models/topics.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  let queryString = "SELECT * FROM topics";
+  const extraQueries = [];
```

Comment:
I'm not sure what you need this for?

Reply:
I thought utilizing an array as the second argument in the query was helpful to prevent injections